Streetcode/Streetcode.XIntegrationTest/ServiceTests/BlobServiceTests/Utils/BlobStorageFixture.cs

```diff
@@ -1,55 +1,56 @@
-﻿using Microsoft.Extensions.Configuration;
+﻿using System.Text;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Options;
 using Newtonsoft.Json;
 using Streetcode.BLL.Services.BlobStorageService;
 using Streetcode.DAL.Entities.Media;
 using Streetcode.DAL.Entities.Media.Images;
 using Streetcode.DAL.Persistence;
-using System.Text;
 
 namespace Streetcode.XIntegrationTest.ServiceTests.BlobServiceTests.Utils
 {
     public class BlobStorageFixture : IntegrationTestBase
     {
-        public readonly BlobService blobService;
         private readonly IOptions<BlobEnvironmentVariables> environmentVariables;
-        public readonly string blobPath;
-        private readonly string blobKey;
-        public StreetcodeDbContext TestDbContext { get; private set; }
 
         public BlobStorageFixture()
         {
-            environmentVariables = Options.Create(new BlobEnvironmentVariables
+            this.environmentVariables = Options.Create(new BlobEnvironmentVariables
             {
-                BlobStoreKey = Configuration.GetValue<string>("Blob:BlobStoreKey"),
-                BlobStorePath = Configuration.GetValue<string>("Blob:BlobStorePath")
+                BlobStoreKey = this.Configuration.GetValue<string>("Blob:BlobStoreKey") !,
+                BlobStorePath = this.Configuration.GetValue<string>("Blob:BlobStorePath") !,
             });
-            blobPath = environmentVariables.Value.BlobStorePath;
-            blobKey = environmentVariables.Value.BlobStoreKey;
+            this.BlobPath = this.environmentVariables.Value.BlobStorePath;
 
-            blobService = new BlobService(environmentVariables); // add repo
-            Directory.CreateDirectory(blobPath);
+            this.BlobService = new BlobService(this.environmentVariables); // add repo
+            Directory.CreateDirectory(this.BlobPath);
         }
 
+        public BlobService BlobService { get; }
+
+        public string BlobPath { get; private set; }
+
+        public StreetcodeDbContext TestDbContext { get; private set; } = null!;
```

Comment:
Better to write this properties before constructor (check in other places in your new code).